modules/ROOT/pages/configure-tls-context-rtf-lb.adoc

```diff
@@ -0,0 +1,54 @@
+= Configure a TLS Context for Anypoint Runtime Fabric
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+All inbound traffic entering Anypoint Runtime Fabric is encrypted using TLS. When enabling inbound traffic, you must provide a valid TLS certificate. When you configure inbound traffic for Runtime Fabric, you can configure the TLS-server by importing a Privacy-Enhance Mail (PEM) or Java Keystore (JKS), or by importing a TLS context from secrets manager (advanced use case).
```

Comment:
I was not sure what PEM and JKS are supposed to modify, since they are file formats: file? truststore? keystore? I've suggested "file," but please correct that if wrong.
```suggestion
All inbound traffic entering Anypoint Runtime Fabric is encrypted using TLS. When enabling inbound traffic, you must provide a valid TLS certificate. When you configure inbound traffic for Runtime Fabric, you can configure the TLS server by importing a Privacy-Enhanced Mail (PEM) or Java Keystore (JKS) file, or by importing a TLS context from the secrets manager (advanced use case).
```